api/app/src/routes/medical/document.ts

```diff
@@ -63,32 +67,25 @@ router.post(
 /** ---------------------------------------------------------------------------
  * GET /document
  *
- * Downloads the specified document for the specified patient.
+ * Fetches the document from S3 and sends a presigned URL
  *
- * @param req.query.patientId Patient ID for which to retrieve document metadata.
- * @param req.query.facilityId The facility providing NPI for the document download.
- * @param req.query.location The document URL.
- * @param [req.query.mimeType] The mime type of the document.
- * @param [req.query.fileName] The file name of the document.
- * @return The document content.
+ * @param req.query.fileName The file name of the document in s3.
+ * @return presigned url
  */
 router.get(
-  "/download",
+  "/downloadUrl",
```

Comment:
Didn't we want to keep the URL abstracted away from the caller, by keeping this `/download` and returning a redirect/302 to the temp signed S3 URL?

Reply:
yeah, but `url` would point to `/document`, so the browser deals with the redirect and download :)